### PR TITLE
feat: 177 add more examples into the docs

### LINKED ISSUE #177: Add more examples to the Cientos docs
### Description

Currently, in the cientos docs there are just a few examples of the components, it will be better to provide examples as in MeshWobbleMaterial to as many components as possible

### Suggested solution

Basically add example to all abstractions, controls, shapes, material and misc. Possible

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/cientos/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/cientos/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://cientos.tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/cientos/issues) that reports the same bug to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
closes #177 

This is ready, but first this one https://github.com/Tresjs/cientos/pull/198 should be merged

### CODE REVIEW

**Review by alvarosabu — CHANGES_REQUESTED** [review]
All of them work but the Precipitation one https://deploy-preview-202--cientos-tresjs.netlify.app/guide/staging/precipitation.html take a look @JaimeTorrealba

**Comment by JaimeTorrealba** [comment]
[Precipitation _ Cientos.webm](https://github.com/Tresjs/cientos/assets/63722373/a1ea8831-179e-4fa5-951a-1bff9cfd0e68)
I don't see problems @alvarosabu 
What's the issue??

**Comment by alvarosabu** [comment]
Screenshot 2023-09-12 at 14 27 02

**Review by alvarosabu — APPROVED** [review]

**Comment by alvarosabu** [comment]
If you give me the ok @JaimeTorrealba we can merge

File: docs/.vitepress/theme/components/ScrollControlsDemo.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <TresPerspectiveCamera :position="[0, 2, 5]" />
+    <Stars :radius="1" />
+    <TresGridHelper :args="[10, 10]" />
+
+    <ScrollControls
+      ref="scRef"
+      v-model="progress"
+      :distance="20"
+      :smooth-scroll="0.1"
+      html-scroll
```
Comment (JaimeTorrealba) [inline docs/.vitepress/theme/components/ScrollControlsDemo.vue:54]: I will change this first @alvarosabu

**Pull request merged**